src/librustc_trans/builder.rs

```diff
@@ -46,6 +50,24 @@ fn noname() -> *const c_char {
 }
 
 impl<'a, 'tcx> Builder<'a, 'tcx> {
+    pub fn entry_block(ccx: &'a CrateContext<'a, 'tcx>, llfn: ValueRef) -> Self {
+        Builder::new_block(ccx, llfn, "entry-block")
+    }
```

Comment:
Hmm, I think it would be nice if this was actually encapsulated in `define_fn` but I'm not sure that actually works out well.